instrumentation/opentelemetry-instrumentation-aiopg/src/opentelemetry/instrumentation/aiopg/aiopg_integration.py

```diff
@@ -34,8 +36,8 @@ class AiopgIntegration(DatabaseApiIntegration):
     async def wrapped_connection(
         self,
         connect_method: typing.Callable[..., typing.Any],
-        args: typing.Tuple[typing.Any, typing.Any],
-        kwargs: typing.Dict[typing.Any, typing.Any],
+        args: tuple[typing.Any, typing.Any],
```

Comment:
Maybe next step would be to standardize typing imports (e.g. `import typing; var: typing.Any` vs `from typing import Any; var: Any`), and also how `typing_extensions` should be used. I usually see two patterns:
- Unconditionally import from `typing_extensions` if it is introduced in a newer Python version. We need to make sure these imports are updated whenever a Python version is dropped.
- Define a typing compat module, where new typing constructs are conditionally imported from `typing` or `typing_extensions`. Easier to update when a Python version is dropped.